configs/next/distros/redhat-8.mk

```diff
@@ -0,0 +1,97 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Distro dependent basic definitions for RedHat Enterprise Server 8
+# =================================================================
+# This Makefile include contains the definitions for the distro being used
+# for the build process. It contains kernel version infos and distro specific
+# sanity checks.
+
+# Informs the distro supported power archs
+AT_SUPPORTED_ARCHS := ppc64le
+
+# You may force the BUILD_IGNORE_AT_COMPAT general condition by distro
+#BUILD_IGNORE_AT_COMPAT := yes
+
+# Kernel version to build toolchain against
+# - As this distro only supports one arch variation (ppc64le), there is no
+#   need to conditionally define these versions based on arch.
+AT_KERNEL := 4.18    # Current distro kernel version for runtime.
+# Previous distro kernel version for runtime-compat.
+AT_OLD_KERNEL :=
+
+# Inform the mainly supported distros
+AT_SUPPORTED_DISTROS := RHEL8
+
+# Inform the compatibility supported distros
+AT_COMPAT_DISTROS :=
+
+# Sign the repository and packages
+AT_SIGN_PKGS := yes
+AT_SIGN_REPO := yes
+AT_SIGN_PKGS_CROSS := no
+AT_SIGN_REPO_CROSS := yes
+
+# ID of GNUPG key used to sign our packages (main/compat)
+AT_GPG_KEYID  := 6976A827
+AT_GPG_KEYIDC := 6976A827
+AT_GPG_KEYIDL := 6976A827
+# ID of GNUPG key used to sign our repositories (main/compat)
+AT_GPG_REPO_KEYID  := 6976A827
+AT_GPG_REPO_KEYIDC := 3052930D
+AT_GPG_REPO_KEYIDL := 6976A827
+
+# Options required by the command to update the repository metadata
+AT_REPOCMD_OPTS := -p -s sha1 --simple-md-filenames --no-database
+
+# Moved here from build.mk since the value for this variable
+# depends on the distro.
+# For a cross build the executables in the toolchain (gcc, ld, etc.)
+# should be built as 32 bit.
+BUILD_CROSS_32 := no
+
+# As some distros have special requirements for configuration upon final
+# AT installation, put in this macro, the final configurations required
+# after the main build and prior to the rpm build
+define distro_final_config
+    echo "nothing to do."
+endef
+
+# Inform the list of packages to check
+ifndef AT_DISTRO_REQ_PKGS
+    AT_CROSS_PKGS_REQ :=
+    AT_NATIVE_PKGS_REQ := libxslt docbook-style-xsl qt5-devel \
+                          autogen-libopts sqlite-devel
+    AT_COMMON_PKGS_REQ := zlib-devel ncurses-devel flex bison texinfo \
+                          createrepo glibc-devel subversion cvs gawk \
+                          rsync curl bc automake libstdc\\+\\+-static \
+                          redhat-lsb-core autoconf bzip2-[0-9] libtool-[0-9] \
+                          gzip rpm-build-[0-9] rpm-sign gcc-c++ imake wget \
+                          docbook2X autoconf-archive libffi-devel python36 \
+                          systemtap-sdt-devel
+    AT_CROSS_PGMS_REQ :=
+    AT_NATIVE_PGMS_REQ :=
+    AT_COMMON_PGMS_REQ := git_1.7 make_4.0
```

Comment:
The `AT_*_PGMS_REQ` variables were used just to document programs that had to be manually built because the packages available in the distro were too old.  I suspect that's the case for git and make.
Do we really need to list them here?  Can't we move them to `AT_COMMON_PKGS_REQ`?

Reply:
Fixed